ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -53,48 +54,48 @@ const (
 	LocalInFileName = "task-definition.json"
 )
 
-// Interface for a local project, holding data needed to convert an ECS Task Definition to a Docker Compose file
-type LocalProject interface {
-	ReadTaskDefinition() error
-	Convert() error
-	Write() error
-	LocalOutFileName() string
-	TaskDefinition() *ecs.TaskDefinition
-	InputMetadata() *converter.LocalCreateMetadata
-}
-
-type localProject struct {
-	context          *cli.Context
-	taskDefinition   *ecs.TaskDefinition
-	localBytes       []byte
-	localOutFileName string
-	inputMetadata    *converter.LocalCreateMetadata
+// LocalProject holds data needed to convert an ECS Task Definition to Docker Compose files.
+type LocalProject struct {
+	context         *cli.Context
+	taskDefinition  *ecs.TaskDefinition
+	baseCompose     []byte
+	overrideCompose []byte
```

Comment:
Nit: I find these two field names to be a bit ambiguous -- the original name `localBytes` was modeled after `composebytes` in [libcompose](https://github.com/aws/amazon-ecs-cli/blob/3777f934adb729b0299b53b9022eeae2ae610715/ecs-cli/vendor/github.com/docker/libcompose/project/context.go#L23). I'm wondering if it makes sense to store both the compose and override bytes together as `[][]byte`, modeling the way `readComposeFiles` works in libcompose: https://github.com/aws/amazon-ecs-cli/blob/3777f934adb729b0299b53b9022eeae2ae610715/ecs-cli/vendor/github.com/docker/libcompose/project/context.go#L23